app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
+end
```

Comment:
xoa file nay no bao loi c ak. "Couldn't find UsersHelper, expected it to be defined in helpers/users_helper.rb"